docs/code-reviews/README.md

```diff
@@ -1,6 +1,6 @@
 # Code Reviews
 
-Developers working on projects should conduct peer code reviews on every pull request (or check-in to a shared branch).
+Code reviews are an essential part of the software development process, offering numerous benefits such as improved code quality, shared knowledge, and early bug detection. However, the effectiveness of these reviews largely depends on the approach and mindset of the participants.
```

Comment:
I like this update. "Why we do it, how it helps you"

Reply:
Open to pushback, but instead of essential, we say it's required.

Reply:
updated from essential to 'required' per feedback.